content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,104 @@
 
-.. image:: media/generate_from_website06.png
-   :align: center
+Contact Forms can be customized for the specific information your team needs, using the free *Form
+Builder* module.
 
-By default, any new contact form will send an email. You can switch to
-lead/opportunity generation in *Change Form Parameters*.
+The free *Form Builder* module can be installed from either the apps page or automatically when
+adding a form to a page in the Website Builder. Adding a whole new form can be useful if you are
+soliciting certain information that needs filtering, such as suggestions, bug reports, etc.
 
-.. note::
-   If the same visitor uses the contact form twice, the second
-   entry will be added to the first lead/opportunity in the chatter.
+.. image:: generate_from_website/form-building-block.png
+   :align: center
+   :alt: Form Builder building blocks
+
+.. tip::
+   Forms can be created from scratch to serve a wide variety of purposes. However, Odoo's default
+   *Contact Us* page is designed to fit most users' needs. Start with the default form and modify
+   from there.
+
+Edit Contact Form fields
+------------------------
+
+In Edit mode on your website, click on any field to start editing it. The following information can
+be edited for each field on the Contact Form:
+
+- **Type**: Choose a custom field option or an existing field. Examples include phone, file
+  upload, language, etc.
+- **Input Type**: Determine the type of entry customers should input. Available options are text,
+  email, telephone, and URL.
+- **Input Placeholder**: Type in an example to guide users how to input information where
+  formatting is important, such as a phone number or email address.
+- **Label Name**: Type in the display name to show users what information is needed from them.
+- **Label Position**: Choose the way the label is aligned with the rest of the form. The label
+  can be hidden, above the field, to the far left of the field, or right adjusted and closer to the
+  field.
+- **Required**: Toggle this option for information that you absolutely need entered.
+- **Hidden**: Toggle this option to hide the field without deleting it.
+- **Shown on Mobile**: Toggle this option to show the field to users on mobile devices.
+
+.. image:: generate_from_website/editable-field-options.png
+   :align: center
+   :alt: Editable field options
 
-Generate leads instead of opportunities
-=======================================
+By default, when a Form is submitted, it sends you an email with the customer's inputted
+information. To have it automatically generate a lead/opportunity instead, edit the Form and select
+**Create an Opportunity** as the Action.
 
-When using a contact form, you should use a qualification step
-before assigning the form to the right sales people. To do so, activate *Leads*
-in CRM settings and refer to :doc:`convert`.
+.. tip::
+   If leads are activated in your CRM settings, selecting **Create an Opportunity** generates a
```

Comment:
This should use `:guilabel:` formatting rather than **bold** text.
```suggestion
   If leads are activated in your CRM settings, selecting :guilabel:`Create an Opportunity` generates a
```